docs/deploy/providers/aws.md

```diff
@@ -14,3 +14,89 @@ import { handler } from './.output/server'
 // Use programmatically
 const { statusCode, headers, body } = handler({ rawPath: '/' })
 ```
+
+# AWS Lambda@Edge
+
+**Preset:** `aws-lambda-edge` ([switch to this preset](/deploy/#changing-the-deployment-preset))
+
+Nitro provides a built-in preset to generate output format compatible with [AWS Lambda@Edge](https://docs.aws.amazon.com/lambda/latest/dg/lambda-edge.html).
+
+The output entrypoint in `.output/server/index.mjs` is compatible with [AWS Lambda@Edge format](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-event-structure.html).
+
+### Deploy using AWS CDK
+
+The following code is an example of deploying a Nuxt3 project to CloudFront and Lambda@Edge with [AWS CDK](https://github.com/aws/aws-cdk). Using this stack, paths under `_nuxt/` (static assets) will get their data from the S3 origin, and all other paths will be resolved by Lambda@Edge.
+
+```ts
+import { spawnSync } from "child_process";
```

Comment:
I think we can auto generate this script to the .output. avoiding to hardcode things like public path to the example.

Reply:
You mean the app build part? I'd like that. Maybe the output could be an object containing the key locations and configs that can be used for cloud specific deployments (for example configuring the S3 bucket and cloudfront caching):

```json
{
  "serverHandler": ".output/server",
  "assets": ".output/assets"
  "public": ".output/public"
}
```

Reply:
Thanks for the review.
This script is not a simple script, it is IaC, so it may be difficult to put it in the .output from a DX perspective.
I think a developer wants more freedom to customize it.

Is the key here that the developer needs to be aware of the public and server paths?

Reply:
Surely developers can always override to customize. We can export smaller utils even for better flexibility.

Reply:
On the other hand. These settings are easily extracted from the nuxt.config.ts since they are either Nuxt's defaults or set in that config. So maybe its not even needed.

Reply:
Yes I can agree your thinking @chris-visser. However to find configuration like nuxt.config.ts or nitro.config.ts and more... is a bit difficult for CDK apps.
Also maybe understood @pi0's thinking. His goal would be a Zero-Config Providers. Certainly that is one of the important features so I will try implemantation for auto generate this script to the .output.